### PR TITLE
geojson 3.1.0 :snowflake:

### PR DESCRIPTION
**Pull request by psteyer**
geojson 3.1.0 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-5180]
- dev_url:        https://github.com/jazzband/geojson/tree/3.1.0
- conda_forge:    https://github.com/conda-forge/geojson-feedstock/blob/main/recipe/meta.yaml
- pypi:           https://pypi.org/project/geojson/3.1.0
- pypi inspector: https://inspector.pypi.io/project/geojson/3.1.0

### Explanation of changes:

- new version number

[PKG-5180]: https://anaconda.atlassian.net/browse/PKG-5180?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,34 +1,52 @@
-{% set version = "2.5.0" %}
+{% set version = "3.1.0" %}
 
 package:
   name: geojson
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/g/geojson/geojson-{{ version }}.tar.gz
```
Comment (ifitchet) [inline recipe/meta.yaml]: For consistency with other recipes?
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,34 +1,52 @@
...
+  description: |
+    This Python library contains, functions for encoding and 
+    decoding GeoJSON formatted data. Classes for all GeoJSON Objects.
+    An implementation of the Python __geo_interface__ Specification
   license: BSD-3-Clause
   license_file: LICENSE.rst
-  summary: 'Python bindings and utilities for GeoJSON'
+  license_family: BSD
+  dev_url: https://github.com/jazzband/geojson
+  doc_url: https://github.com/jazzband/geojson/blob/main/README.rst
```
Comment (ifitchet) [inline recipe/meta.yaml:52]: Of limited interest, the GH repo's `homepageUrl` points at PyPI (you can see it at the top right) and the PyPI page's Homepage points back to GH... _caveat automaton!_

**Review by ifitchet — APPROVED** [review]
`source.url` could be improved (see suggestion).

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,34 +1,52 @@
...
   url: https://pypi.io/packages/source/g/geojson/geojson-{{ version }}.tar.gz
-  sha256: 6e4bb7ace4226a45d9c8c8b1348b3fc43540658359f93c3f7e03efa9f15f658a
+  sha256: 58a7fa40727ea058efc28b0e9ff0099eadf6d0965e04690830208d3ef571adac
 
 build:
   number: 0
-  noarch: python 
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  skip: true  # [py<37]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: It doesn't support py313+, see https://github.com/jazzband/geojson/blob/3.1.0/setup.py#L21

**Review by skupr-anaconda — APPROVED** [review]
LGTM but py313+ won't work

File: recipe/meta.yaml (hunk 3 above)
Comment (ifitchet) [reply recipe/meta.yaml]: A bit annoying it didn't say that on [L41](https://github.com/jazzband/geojson/blob/3.1.0/setup.py#L41)!

**Pull request merged**